index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Microverse Concert</title>
+  </head>
+  <body>
+    <section class="navigation">
+      <nav class="top-bar">
+        <i class="fa fa-facebook"></i>
+        <i class="fa fa-instagram"></i>
+        <i class="fa fa-youtube"></i>
+        <a href="#">Login</a>
+      </nav>
+      <nav class="flex-nav">
+        <div class="menu-open">
+          <img src="assets/img/ic_menu.svg" alt="" />
+        </div>
+        <div class="menu">
+          <a href="index.html" class="logo"
+            ><img src="assets/img/logo.png" alt=""
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
+      </nav>
+    </section>
+    <header>
+      <p class="hello">"Hello Micronauts!"</p>
+      <h1 class="title">Microverse Concert Music for Developers 2021</h1>
+      <div class="intro">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed facilis porro
+          maiores, rerum vitae.
+        </p>
+      </div>
+      <p class="date">05(FRI)-07(SUN) DEC 2021</p>
+      <p class="at">@ YouTube: Microverse Official Channel</p>
+    </header>
```

Comment:
- [ ] As I previously stated, kindly add the `header` tag as a container for the navigation bar as this helps with semantic correctness and overall web page accessibility. This means the headline should be grouped with a `section` and then added to the `main` tag.
- [ ] As this is a headline, kindly use a `section` tag to group it. 
- [ ] The `Hello Micronauts` is a subheading and should be grouped using a `hx` tag instead of a `p` tag.
- [ ] Also, increase the font size of the headline intro to improve the readability and overall webpage aesthetics.